src/Console/Coverage.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
Since this is neither a CakePHP shell nor a task, I find it somewhat confusing to find this file directly within the Console folder.
I see, there is already an [Installer.php](https://github.com/cakephp/app/blob/3.3.6/src/Console/Installer.php), which is also a Composer related script.

I would except to find such files within a sub folder named Composer, for example.